sp/src/game/shared/hl2/basehlcombatweapon_shared.cpp

```diff
@@ -412,6 +456,130 @@ Activity CBaseHLCombatWeapon::ActivityOverride( Activity baseAct, bool *pRequire
 
 	return BaseClass::ActivityOverride( baseAct, pRequired );
 }
+
+//-----------------------------------------------------------------------------
+// Purpose: Drop/throw the weapon with the given velocity.
+//-----------------------------------------------------------------------------
+void CBaseHLCombatWeapon::Drop( const Vector &vecVelocity )
+{
+	m_iClip1 /= 2;
```

Comment:
Where did this clip half come from? If this is for dual wielding, doesn't it need to be behind a check? I might be misunderstanding

Reply:
You're right, this needed to be behind a check. Thanks for pointing that out